src/main/test/CommandHandlerTests.cpp

```diff
@@ -74,21 +74,87 @@ TEST_CASE("transaction envelope bridge", "[commandhandler]")
 
     SECTION("new-style transaction v0")
     {
-        for_all_versions(*app, [&]() {
-            closeLedgerOn(*app, 2, 1, 1, 2017);
+        SECTION("valid without timebounds")
```

Comment:
These four sections look like they're the same except for the timebounds and the result. I think it'd be a good idea if they were de-duplicated using a lambda.

Reply:
I'll take a crack at making that change. Thanks for the feedback!

Reply:
@sisuresh I moved repetitive logic into a lambda in f0b52d9.

However, I think the tests are a bit harder to read. I suspect this is an example of making tests too<sup>[1]</sup> DRY<sup>[2]</sup>. I'm happy to keep the change though if this is more consistent with how stellar-core tests are written.

Let me know what you think. If you're happy with this, this PR is ready for final review and merge.

[1]: https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html
[2]: https://thoughtbot.com/blog/the-case-for-wet-tests

Reply:
The other lambda driven tests in this file keep the `closeLedgerOn` and `REQUIRE` outside the lambda which is why I'd taken that approach.

I pushed df61e313 which moves as much as possible into lambda. For ease of reading without context jumping and evolving tests over time I think a repetitive approach is preferable, but I'm contributing to your code base here, and this is concise for sure.

Reply:
This type of approach is what I prefer and what we generally do now, but you bring up a good point on readability. I would be curious to get some other opinions because I do believe the core tests in general aren't always easy to read. I do think this PR in it's current state is good though.

Reply:
I'm happy with this as is if you are. It is more consistent with other tests in the file than the PR was prior to these changes.

Reply:
Yeah I think this is good to merge. Do you mind squashing your commits?

Reply:
😱 ... done.